src/elm/Page/Shop/Viewer.elm

```diff
@@ -484,13 +470,6 @@ viewCard session card model =
                       else if card.sale.units <= 0 && card.sale.trackStock == True then
                         div [ class "flex -mx-2 md:justify-end" ]
                             [ button
-                                [ class "button button-secondary w-1/5 mx-2"
-                                , onClick (ClickedAsk card.sale)
-                                ]
-                                [ Avatar.view (getIpfsUrl session) card.sale.creator.avatar "h-6 w-6 mr-2"
-                                , text_ "shop.ask"
-                                ]
-                            , button
                                 [ disabled True
                                 , class "button button-disabled w-4/5 mx-2 md:w-2/5"
```

Comment:
Hi @namoscagnm  no worries about showing you so our application uses tailwindcss for styling which you can find out more about here https://tailwindcss.com/docs/margin/#app this page will also show you how to set margins however the easiest way for you to do this would be to add this rule `sm:mx-auto` to this line(#474) and (#491) what this does is it adds an auto margin on smaller devices to know more about using this you would need to look into tailwind but you can complete this without doing that.
Please let me know if I can help further